packages/ui/src/components/modals/ChangeMultisig.tsx

```diff
@@ -461,9 +454,10 @@ const ChangeMultisig = ({ onClose, className }: Props) => {
                 text={errorMessage}
               />
             )}
-            {currentStep === 'summary' && <Button onClick={onGoBack}>Back</Button>}
+            {currentStep === 'summary' && <Link onClick={onGoBack}>Back</Link>}
```

Comment:
I find that having a mix of buttons and link is ugly. I'll be using a `Button` here as well.